nrfx_config.h

```diff
@@ -48,6 +48,44 @@
 #define NRFX_CLOCK_ENABLED 1
 #endif
 
+#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
+#if defined(CONFIG_SOC_SERIES_NRF91X) || defined(CONFIG_SOC_SERIES_NRF53X)
+#define NRFX_CLOCK_CONFIG_LF_SRC 1
+#else
+#define NRFX_CLOCK_CONFIG_LF_SRC 0
+#endif
+#elif CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL
```

Comment:
It should be `#ifdef`, for consistency. And I don't think we need to use here `#else` when checking those options. They are choice options, so Kconfig should take care for defining only one of them.
Thus, I suggest to use:
```
#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
#if defined(CONFIG_SOC_SERIES_NRF91X) || defined(CONFIG_SOC_SERIES_NRF53X)
#define NRFX_CLOCK_CONFIG_LF_SRC 1
#else
#define NRFX_CLOCK_CONFIG_LF_SRC 0
#endif
#endif /* CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC */

#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL
...
```